.changelog/1564.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:feature
+New server platform - ECS: Use `--platform=ecs` to install waypoint server to
```

Comment:
```suggestion
serverinstall/ecs: New server platform - ECS: Use `--platform=ecs` to install waypoint server to
```

Reply:
I thought about this one and felt that the package name (`serverinstall`) was too niche to be immediately obvious to causal readers what it meant. I can accept this if you feel strongly about it though

Reply:
Hmnm good point. We do need something here. In the past I've used `serverinstall`. I don't know if `plugin/ecs` makes sense, but maybe `install/ecs`?